Blockzilla/FullScreen.js

```diff
@@ -0,0 +1 @@
+!function(e){var t={};function n(r){if(t[r])return t[r].exports;var u=t[r]={i:r,l:!1,exports:{}};return e[r].call(u.exports,u,u.exports,n),u.l=!0,u.exports}n.m=e,n.c=t,n.d=function(e,t,r){n.o(e,t)||Object.defineProperty(e,t,{enumerable:!0,get:r})},n.r=function(e){"undefined"!=typeof Symbol&&Symbol.toStringTag&&Object.defineProperty(e,Symbol.toStringTag,{value:"Module"}),Object.defineProperty(e,"__esModule",{value:!0})},n.t=function(e,t){if(1&t&&(e=n(e)),8&t)return e;if(4&t&&"object"==typeof e&&e&&e.__esModule)return e;var r=Object.create(null);if(n.r(r),Object.defineProperty(r,"default",{enumerable:!0,value:e}),2&t&&"string"!=typeof e)for(var u in e)n.d(r,u,function(t){return e[t]}.bind(null,u));return r},n.n=function(e){var t=e&&e.__esModule?function(){return e.default}:function(){return e};return n.d(t,"a",t),t},n.o=function(e,t){return Object.prototype.hasOwnProperty.call(e,t)},n.p="",n(n.s=31)}({31:function(e,t,n){e.exports=n(32)},32:function(e,t){var n=!!(document.fullscreenEnabled||document.webkitFullscreenEnabled||document.mozFullScreenEnabled||document.msFullscreenEnabled),r=void 0!==HTMLVideoElement.prototype.webkitEnterFullscreen;n||!r||/mobile/i.test(navigator.userAgent)||(HTMLElement.prototype.requestFullscreen=function(){if(void 0!==this.webkitRequestFullscreen)return this.webkitRequestFullscreen(),!0;if(void 0!==this.webkitEnterFullscreen)return this.webkitEnterFullscreen(),!0;var e=this.querySelector("video");return void 0!==e&&(e.webkitEnterFullscreen(),!0)},Object.defineProperty(document,"fullscreenEnabled",{get:function(){return!0}}),Object.defineProperty(document.documentElement,"fullscreenEnabled",{get:function(){return!0}}))}});
```

Comment:
Is this this [FullScreenHelper.js](https://github.com/mozilla-mobile/firefox-ios/blob/main/Client/Frontend/UserContent/UserScripts/AllFrames/WebcompatAtDocumentStart/FullscreenHelper.js) from Firefox iOS?

Would you mind replacing this minified version with the original? I like using minified code, but I think we should start with the normal one and then maybe later think of a way to automate the minification as a build step.

This script is small enough that the impact of minification is pretty small I think so I'm not worried about performance or size.

Reply:
Ok, I'll replace it with the original.